lib/live_select/component.html.heex

```diff
@@ -8,10 +8,8 @@
   data-field={@field.id}
   data-debounce={@debounce}
 >
-  <%= if @mode == :tags && Enum.any?(@selection) do %>
-    <div class={
-      class(@style, :tags_container, @tags_container_class, @tags_container_extra_class)
-    }>
+  <div class={class(@style, :tags_container, @tags_container_class, @tags_container_extra_class)}>
+    <%= if (@mode == :tags || @mode == :quick_tags) && Enum.any?(@selection) do %>
```

Comment:
I prefer `@mode in [:tags, :quick_tags]` , I think it's more readable (nitpick, I know)

Reply:
Agree!